test/unittests/execute_control_test.cpp

```diff
@@ -1,22 +1,30 @@
 #include "execute.hpp"
 #include "parser.hpp"
 #include <gtest/gtest.h>
+#include <test/utils/asserts.hpp>
 #include <test/utils/hex.hpp>
 
 using namespace fizzy;
 
 TEST(execute_control, unreachable)
 {
-    Module module;
-    module.codesec.emplace_back(Code{0, {Instr::unreachable, Instr::end}, {}});
-
-    const auto [trap, ret] = execute(module, 0, {});
+    /*
+    (func unreachable)
+    */
+    const auto wasm = from_hex("0061736d01000000010401600000030201000a050103000000");
 
-    ASSERT_TRUE(trap);
+    const auto [trap, ret] = execute(parse(wasm), 0, {});
+    EXPECT_TRUE(trap);
 }
 
 TEST(execute_control, nop)
 {
+    /*
+    (func
+      nop
+    )
+    */
+    const auto wasm = from_hex("0061736d01000000010401600000030201000a05010300010b");
     Module module;
     module.codesec.emplace_back(Code{0, {Instr::nop, Instr::end}, {}});
```

Comment:
This case is weird, has wasm and module.

Reply:
Fixed.